e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -148,6 +148,29 @@ describe('Upload Files', () => {
         });
     });
 
+    it('MM-T341 Download link on preview - Image file (non SVG)', () => {
+        ['small-image.png', 'image-file.jpg', 'image-file.bmp', 'image-file.gif', 'image-file.tiff'].forEach((image) => {
```

Comment:
`.ico` and `.webp` files don't have image thumbnail preview, but SVG does. Could someone please confirm if SVG is intended to be included in this test?

Reply:
Thanks for checking @clarmso. The file types you have chosen in sufficient. This test specifically indicates non SVG files. Hence we can ignore the SVG files.